source/contentprovider.d

```diff
@@ -367,7 +383,7 @@ class ContentProvider
 	  over the whole content, regardless of language. Content
 	  doesn't guarantee any order.
 	+/
-	LanguageMeta getMeta(string language) const
+	const(LanguageMeta) getMeta(string language) const
```

Comment:
This was necessary as LanguageMeta is no longer a simple struct that will be copied entirely and thus is `const` automatically by definition.